File: Leetcode Python/Max Consecutive Ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        z=len(nums)
        if z==1:
            return nums[0]
        t,c=0,0
        for i in range(z):
            if nums[i]==1:
                c+=1
            else:
                t=max(t,c)
                c=0
        t=max(t,c)
        return t

File: Leetcode Cpp/Max Consecutive Ones.cpp
class Solution {
public:
        Solution() {
        ios::sync_with_stdio(0);
        cin.tie(0);
    }
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int t=0,c=0;
        for(int i:nums){
            if(i==1){
                c++;
            }
            else{
                t=max(t,c);
                c=0;
            }
        }
        t=max(t,c);
        c=0;
        return t;
    }
};